src/main/java/io/projectreactor/DocUtils.java

```diff
@@ -122,6 +122,42 @@ else if (reqUri.contains("/kdoc-api/")) {
 		return url;
 	}
 
+	static boolean hasKDoc(String module, String version) {
+		if (("core".equals(module)
```

Comment:
nit: WDYT about `switch` on module's name + an early return? for the readability